ci-operator/step-registry/ipi/deprovision/deprovision/in-bastion/ipi-deprovision-deprovision-in-bastion-commands.sh

```diff
@@ -60,24 +73,72 @@ BASTION_SSH_USER=$(<"${SHARED_DIR}/bastion_ssh_user")
 SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
 REMOTE_DIR="/home/${BASTION_SSH_USER}"
 REMOTE_INSTALL_DIR="${REMOTE_DIR}/installer/"
-REMOTE_ENV_FILE="${REMOTE_DIR}/remote_env_file"
+REMOTE_ENV_FILE="/tmp/remote_env_file"
+
+# Prepare credentials
+# Update here to support intallation in bastion on different platforms
+case "${CLUSTER_TYPE}" in
+azure4|azuremag|azure-arm64)
```

Comment:
@jinyunma If I remember right, `cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-managed-identity-deprovision` does not need sp for deprovision, pls help review to confirm.

Reply:
`azure_managed_identity_osServicePrincipal.json` is required to destroy cluster, but we already uploaded credential files into bastion host in step [ipi-install-install-in-bastion](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/in-bastion/ipi-install-install-in-bastion-commands.sh#L146-L158), I think there is no need to upload again here.

Reply:
sorry, I didn't read the description of this PR.
>  ${REMOTE_DIR} on bastion host is cleaned up finally during the step ipi-install-install-in-bastion

if so, these steps in the script are required.

Reply:
hmm, how about to upload credential files into dir `${REMOTE_DIR}/secrets` for example in `ipi-install-install-in-bastion` and clean up this folder in the end of this step? Then only credentials files need to be recovered in `ipi-deprovision-deproivsion-in-bastion`.

Reply:
Agree. That will save us to recover a lot of things, including installer binary.